app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -916,6 +932,33 @@ impl Node {
             color::Lcha::transparent()
         }
     }
+
+    fn node_bounding_box(
```

Comment:
Why is this an associated function? We are in module `node` - why an associated function to the `Node` structure is better than just a module-function in this case?

Reply:
Also, this is associated function to `Node`, while the function `position_of_visualization_relative_to_position_of_node_of_given_size` is associated to `NodeModel` – what is the reason for this design?